internal/common/conversion/import_state.go

```diff
@@ -10,7 +10,26 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/resource"
 )
 
-func ImportStateProjectIDClusterName(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+type ClusterImportAttrNames struct {
```

Comment:
It is a bit controversial struct, but it is necessary to:
1. Support when `cluster_name` is defined as `name` in the ResourceSchema
2. Have a single place where the "defaults" are defined (invoked in `testutil/acc/import_state.go` also

Reply:
changed in 49744fbe80fc1cebc33651fcf22c380e85cf6094